src/main/java/com/intellias/basicsandbox/config/actuator/ActuatorConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package com.intellias.basicsandbox.config.actuator;
+
+import org.springframework.boot.actuate.web.exchanges.HttpExchangeRepository;
+import org.springframework.boot.actuate.web.exchanges.InMemoryHttpExchangeRepository;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ActuatorConfiguration {
+    @Bean
+    public HttpExchangeRepository httpTraceRepository() {
+        return new InMemoryHttpExchangeRepository();
```

Comment:
What's a purpose of this bean?

Reply:
For enabling the `/httpexchanges` endpoint in Spring Boot 3, you have to create a bean of the `HttpExchangeRepository` type.
I'll add a comment about it.